applovin_max/lib/applovin_max.dart

```diff
@@ -313,6 +313,13 @@ class AppLovinMAX {
     });
   }
 
+  /// Sets a list of the ad units for the SDK to initialize only those networks.
```

Comment:
We should note that this should be set before initializing the SDK. 
```suggestion
  /// Sets a list of the ad units for the SDK to initialize only those networks.
  /// Should be set before initializing the SDK.
```